modules/generators/content/password_file/secgen_metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+
+<generator xmlns="http://www.github/cliffe/SecGen/generator"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://www.github/cliffe/SecGen/generator">
+
+  <name>Password List File Generator</name>
+  <author>Jack Biggs</author>
+  <module_license>MIT</module_license>
+  <description>Creates a file in whcih login are contained for 10 websites.</description>
```

Comment:
Why 10? I suggest you allow however many are fed in (with a default of "nothing2seehere") via the passwords parameter, and use that many websites. For example, if "pass123" and "password123" were fed in the file would have two random websites using those passwords.